README.md

```diff
@@ -1,2 +1,307 @@
+Provision the following yaml to define the overlay which will configure the
+secondary attachment for the KubeVirt VMs.
+
+Please refer to the OVN-Kubernetes user documentation for details into each of
+the knobs.
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: flatl2
+spec:
+    config: |2
+    {
+        "cniVersion": "0.3.1",
+        "name": "flatl2",
+        "netAttachDefName": "default/flatl2",
+        "topology": "layer2",
+        "type": "ovn-k8s-cni-overlay"
+    }'
+EOF
+```
+
+The above example will configure a cluster-wide overlay without a subnet
+defined. This means the users are responsible for providing an alternative way
+to configure the VM's IP addresses for the overlay network - either manually
+configuring those via `cloud-init`, of by deploying a DHCP server in the overlay
+network. In [deploying the DHCP server](#deploying-the-dhcp-server), we show a
+simple way of achieving the latter.
+
+It is also worth to point out the value of the netAttachDefName attribute must
+match the <namespace>/<name> of the surrounding `NetworkAttachmentDefinition`
+object.
+
+## Deploying the DHCP server
+Once the network definition is provisioned, the network administrator should
+deploy the DHCP server. Use the following yaml for that:
+```bash
+cat <<EOF | kubectl apply -f -
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dhcp-server-conf
+data:
+  dhcpd.conf: |
+    authoritative;
+
+    default-lease-time 86400;
+    max-lease-time 86400;
+
+    subnet 192.168.123.0 netmask 255.255.255.0 {
+            range 192.168.123.2 192.168.123.254;
+            option broadcast-address 192.168.123.255;
+    }
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dhcp-server
+  labels:
+    app: dhcp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dhcp
+  template:
+    metadata:
+      labels:
+        app: dhcp
+      annotations:
+        k8s.v1.cni.cncf.io/networks: '[
+          {
+            "name": "flatl2",
+            "ips": ["192.168.123.1/24"]
+          }
+        ]'
+    spec:
+      containers:
+      - name: server
+        image: quay.io/mduarted/dhcpd
+        args: ["-4", "-f", "-d", "--no-pid", "-cf", "/etc/dhcp/dhcpd.conf"]
+        securityContext:
+          runAsUser: 1000
+          privileged: true
+        volumeMounts:
+        - name: multus-daemon-config
+          mountPath: /etc/dhcp
+          readOnly: true
+      volumes:
+      - name: multus-daemon-config
+        configMap:
+          name: dhcp-server-conf
+          items:
+          - key: dhcpd.conf
+            path: dhcpd.conf
+EOF
+```
+
+The yaml above provisions a deployment (with a single replica) that will ensure
+the DHCP server is always up. Keep in mind this is a simple example and should
+**not** be used in production - there is a single point of failure (the DHCP
+server), and the leases file is not persisted outside the pod.
```

Comment:
Is the blogpost targeting production environments? If so, how should it be dine in production?

Reply:
Personally, I would use static IP addresses. 

If people want to use a container based DHCP server, well, they're a bit on their own. My goal is to show showing potential via a toy.

Reply:
I am investigating how to use storage in openshift to get the claims re-used by the different deployment pods.